pkg/admission/chain.go

```diff
@@ -23,15 +23,20 @@ type chainAdmissionHandler []Interface
 
 // NewFromPlugins returns an admission.Interface that will enforce admission control decisions of all
 // the given plugins.
-func NewFromPlugins(client clientset.Interface, pluginNames []string, configFilePath string) Interface {
+func NewFromPlugins(client clientset.Interface, pluginNames []string, configFilePath string, plugInit PluginInitializer) (Interface, error) {
```

Comment:
Can you split this change specifically into its own commit - this appears to be the only API change here, and it's better it be separate.